examples/applications/plot_model_complexity_influence.py

```diff
@@ -16,17 +16,21 @@
 
 We will model the complexity influence on three different estimators:
     - :class:`~sklearn.linear_model.SGDClassifier` (for classification data)
-      which implements stochastic gradient descent learning;
+      which implements stochastic gradient descent learning.
+      See example for comparison with different penalties:
+      :ref:`sphx_glr_auto_examples_linear_model_plot_sgd_penalties.py`
 
     - :class:`~sklearn.svm.NuSVR` (for regression data) which implements
-      Nu support vector regression;
+      Nu support vector regression. See example of its application:
+      :ref:`sphx_glr_auto_examples_svm_plot_svm_regression.py`
 
     - :class:`~sklearn.ensemble.GradientBoostingRegressor` builds an additive
       model in a forward stage-wise fashion. Notice that
       :class:`~sklearn.ensemble.HistGradientBoostingRegressor` is much faster
       than :class:`~sklearn.ensemble.GradientBoostingRegressor` starting with
       intermediate datasets (`n_samples >= 10_000`), which is not the case for
-      this example.
+      this example. See detailed example:
+      :ref:`sphx_glr_auto_examples_ensemble_plot_gradient_boosting_regression.py`
```

Comment:
Placing this example in the doc-string for [GradientBoostingRegressor](https://scikit-learn.org/1.5/modules/generated/sklearn.ensemble.GradientBoostingRegressor.html#sklearn.ensemble.GradientBoostingRegressor) might be more suitable.

Reply:
Hi @virchan, thanks for the suggestion! After checking, the reference to the example is already correctly placed in the `GradientBoostingRegressor` doc-string, so no additional changes are necessary.